docs/using-template-repos-for-assignments.md

```diff
@@ -0,0 +1,48 @@
+Importing starter code for assignments is a very useful feature and with GitHub recently shipping the template repositories, we can leverage them in Classroom to make importing starter code much easier and faster!
+
+### How to use template repositories for an assignment?
+
+#### Before you begin
+To use template repositories for assignments, you'll need the following:
+
+* Set-up a GitHub classroom
+* An exercise (a repository that you have access to, which contains documentation, starter code, tests, or anything else your students need to begin work on an assignment).
+
+#### Get started
+
+Once you log in to [GitHub Classroom](https://classroom.github.com/), select your classroom and then click on **New Assignment**. Template repositories work for both **Individual Assignments** and **Group Assignments**. So you can pick either of the two for your assignment. You can then provide details for the assignment. Once you've filled out the details, the **Optional** section has a search bar for finding your starter code repository. When you search for your starter code repository, you will now get two options:
```

Comment:
`Template repositories work for both **Individual Assignments** and **Group Assignments**. So you can pick either of the two for your assignment.`

Second sentence seems repetitive/redundant

Reply:
Can you add a commit suggestion for this?  it's easier to update the doc from here 😄